src/components/TextLink.js

```diff
@@ -32,19 +32,29 @@ const defaultProps = {
 
 const TextLink = (props) => {
     const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
+
+    const handlePress = (e) => {
+        e.preventDefault();
+        if (props.onPress) {
+            props.onPress();
+            return;
+        }
+
+        Linking.openURL(props.href);
+    };
+
     return (
         <Text
             style={[styles.link, ...additionalStyles]}
             accessibilityRole="link"
             href={props.href}
-            onPress={(e) => {
-                e.preventDefault();
-                if (props.onPress) {
-                    props.onPress();
+            onPress={handlePress}
+            onKeyDown={(e) => {
```

Comment:
Please move this callback function to a function defined above like you have the other one.

Reply:
done

Reply:
Hm, this is back again. Is it still possible to move this up to be a callback like the `openLink` method? Maybe call it something like `openLinkIfEnterKeyPressed`. I know that's verbose, but it at least gets the point across.